doc/md/guides/quickstart.mdx

```diff
@@ -657,7 +687,14 @@ root, `projects/cert-manager.cue`
 5. We define Helm values in CUE to take advantage of strong type checking and
 manage multiple Helm charts consistently with platform wide values.
 
-Let's take a look at the BuildPlan this CUE configuration defines.
+Every component modeled within a Holos Platform will have two files working in
+tandem to generate a BuildPlan and achieve our goal of rendering Kubernetes
+manifest files:
+
+  - The platform specification file near the root of the tree (`projects/cert-manager.cue`) that registers the Component.
```

Comment:
This should be `platform/cert-manager.cue`.  Components are registered with the platform in the `platform/` directory.